iphone/Classes/TiFile.m

```diff
@@ -63,7 +61,7 @@ - (unsigned long long)size
 
 - (id)blob
 {
-  return [[[TiBlob alloc] _initWithPageContext:[self pageContext] andFile:path] autorelease];
+  return [[TiBlob alloc] _initWithPageContext:[self pageContext] andFile:_path];
```

Comment:
Why autorelease removed? will it not cause memory leak?

Reply:
I moved the whole class to ARC

Reply:
Cool. Can you once check your test case in ticket. It is giving error for testRecursiveDirectories() and for others its giving 0 but you have mentioned it should be Double. Am I missing anything? Thanks!

Reply:
I think there are some issues with the change in general, some paths are released too early. Will put into progress again for now.